spec/services/atmosphere/flavor_manager_spec.rb

```diff
@@ -3,81 +3,81 @@
 
 describe Atmosphere::FlavorManager do
 
-  let(:cs1) { create(:openstack_compute_site) }
-  let(:cs2) { create(:amazon_compute_site) }
-
-  describe 'CS flavors' do
-    context 'when creating a new cloud site' do
-      it 'registers cloud sites' do
-        expect(cs1.virtual_machine_flavors.count).to eq 0
-        expect(cs2.virtual_machine_flavors.count).to eq 0
+  let(:t1) { create(:openstack_tenant) }
+  let(:t2) { create(:amazon_tenant) }
+
+  describe 'T flavors' do
+    context 'when creating a new tenant' do
+      it 'registers tenants' do
+        expect(t1.virtual_machine_flavors.count).to eq 0
+        expect(t2.virtual_machine_flavors.count).to eq 0
       end
     end
 
-    context 'when populating a cloud site with flavors' do
-      let!(:flavor6_ost) { create(:virtual_machine_flavor, compute_site: cs1, id_at_site: "6", flavor_name: "foo") }
-      let!(:flavor6_aws) { create(:virtual_machine_flavor, compute_site: cs2, id_at_site: "m1.medium", flavor_name: "foo") }
+    context 'when populating a tenant with flavors' do
+      let!(:flavor6_ost) { create(:virtual_machine_flavor, tenant: t1, id_at_site: "6", flavor_name: "foo") }
+      let!(:flavor6_aws) { create(:virtual_machine_flavor, tenant: t2, id_at_site: "m1.medium", flavor_name: "foo") }
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.